Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,46 @@
+
+![pseudo code 2](https://user-images.githubusercontent.com/110966922/202583040-c28e27fe-c175-4287-a8cb-cefafbfe327b.png)
+
+Shown is a schematic of the Arduino Mega 2560 Rev 3.
+![CAPSTONE 1 Microcontroller Schematic](https://user-images.githubusercontent.com/118767661/204694703-fcaa5b67-bf5d-469f-a134-dae06493fe5f.png)
+
+## ANALYSIS OF CHOSEN COMPONENT(S)
+The team has chosen to look at the Arduino Mega 2560 Rev 3 and Arduino USB Shield Host as the solutions.  These components, in conjunction, will satisfy the constraints and jobs of this subsystem.  Following are the corresponding fulfillment of the constraints:
+
+1. The Arduino Mega 2560 Rev 3 falls within the preferred range of $20-$60 at a current price of $48.40 (as of 11-8-22).  The USB Host Shield component has a price of $20.99 as of the same date.   
+2. 4 UART serial hardware ports are present on the board.  SPI is supported as well with included libraries.  As an alternative later, Two-Wire Interface (TWI) is also supported.  These enable the Mega to issue instructions and exchange data between other components and subsystems (Communication).
+3. The USB Host Shield enables the use of USB devices such as a Flash drive for external storage and communication. The Arduino Mega does not support USB as a standalone component.
+4. There are 54 digital input/output pins (of which 15 are PWM outputs) and 16 analog inputs.  This meets the needs for at least 15 of each (digital and analog input pins).  The other 24 digital pins may also be assigned special purposes, so the excess will be used.  
+5. The code is currently of an unknown size, so it is necessary to aim slightly higher in terms of needed processing memory.  The Arduino Mega has 256 KB of Flash, 8 KB of SRAM, and 4 KB of EEPROM.  The SRAM and Flash memory are satisfactory for the subsystem’s needs.
+6. Algorithmic measurement of the battery life is supported.  The board will be able to read the voltage levels coming from the battery by taking a reading from a voltage divider located in the power subsystem.  This reading can then be used as a reference initially and then measured against the incoming voltage from said point continuously. Running an algorithm based on the battery’s expected output vs current output will determine the battery’s capacity.
```

Comment:
I don't agree with this method. Look into it a bit more and you'll see that battery monitoring isn't so simple.

Reply:
Battery monitoring has been made a conjoined effort with the Power subsystem.  This method does work, but would you say that measuring the miliamp-hour value is more accurate?  This would require measuring the voltage across a load resistor and dividing by the known resistance of the resistor to find discharge current.  The current is then multiplied against a time variable.  This method was also considered.